app/managers.py

```diff
@@ -0,0 +1,37 @@
+import sqlite3
+
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self.connection = sqlite3.connect(".../cinema.sqlite")
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self.connection.execute(
+            "INSERT INTO actors (first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self.connection.commit()
+
+    def all(self) -> list:
+        actors = self.connection.execute(
+            "SELECT * FROM actors"
```

Comment:
It's better to follow SQL style guidelines even while writing raw SQL in Python. Therefore I'd suggest to start FROM clause from new line.